cmd/kubevirt-cloud-controller-manager/main.go

```diff
@@ -43,6 +43,10 @@ func main() {
 		klog.Fatalf("unable to initialize command options: %v", err)
 	}
 
+	// To remove role binding to configmap kube-system:extension-apiserver-authentication-reader
+	// cluster are working fine without it
+	ccmOptions.Authentication.SkipInClusterLookup = true
```

Comment:
what behavior in the cloud provider interface actually needs this? Is there a way we can explicitly disable the behavior entirely?

Reply:
We don't need that option. We found why you had a problem with CCM trying to reach the config map on the infra cluster.
As described here https://github.com/kubevirt/cloud-provider-kubevirt/pull/141#issuecomment-1231532936.